config/base/kustomization.yaml

```diff
@@ -1,14 +1,9 @@
 resources:
```

Comment:
There's some GH glitch that is preventing me from putting this comment in [the right place](https://github.com/Kong/kubernetes-ingress-controller/pull/1682/files/99578cb83ec978d365161043ef3a30670e008899#diff-cf639963acef4d0f159a9cea1711b7fcea18507082a04a3e657735d8f358e0c7R17-R18). This comment really refers to the `prometheus.io/*` annotations in the KIC pod template in `base/kong-ingress-dbless.yaml`.

With the `PodMonitor` living in a separate kustomization, I'm inclined to think that these annotations should either go away altogether from `base`, or at least be put in a separate patch, similarly to the `PodMonitor` kustomization.

In practice: either drop these completely (and tell the user to use the prometheus operator or write their own annotation-based prom target discovery wiring), or make these similar to the aforementioned PodMonitor kustomization.

What do you think?

Reply:
Hard to say whether this was providing some non-operator functionality before/whether removing this would reduce default functionality in some environments. These annotations are no longer mentioned (or were never mentioned) in official Prometheus docs, and I'm not sure exactly what in Prometheus or supporting tooling code looked for them.

Have removed them since they are easy enough to re-add.